nixos/tests/unbound.nix

```diff
@@ -63,6 +63,7 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
           enable = true;
           settings = {
             server = {
+              define-tag = [ "local" "test" ];
               interface = [ "192.168.0.1" "fd21::1" "::1" "127.0.0.1" ];
               access-control = [ "192.168.0.0/24 allow" "fd21::/64 allow" "::1 allow" "127.0.0.0/8 allow" ];
```

Comment:
Shouldn't we also add an `access-control-tag` statement that uses the tags to ensure that the config is valid? If I read the bug report right then the issue only occurs when the tag is being "used" before it is defined.

Reply:
Yes, that's why I define these tags before anything else.